### DISCUSSION TITLE
blog/2023/09/linux-is-a-barrier-for-developers/

### DISCUSSION THREAD

**Discussion by giscus** [Ast3risk-ops/blog#15] [Giscus]
# blog/2023/09/linux-is-a-barrier-for-developers/

Here’s why.

https://blog.asterisk.lol/blog/2023/09/linux-is-a-barrier-for-developers/

**Comment by liluzibird** [comment]
Man, I read through this blog post and I think you explained your opinions pretty well. However, I don't know why you think Linux in its current state isn't usable compared to Windows. I along with a couple other friends I know IRL are using Linux as our primary OS. I must agree that there isn't a simple universal package out there that is perfect, however, I know that as long as there is a popular piece of software out there, there will be some kind soul that will repackage and a community to maintain it (See [Synology Drive Client](https://aur.archlinux.org/packages/synology-drive)). Even on Arch Linux (btw) and the ancient X11 graphics system you state, and I have not encountered any issues. Games run just fine, as long as there isn't invasive anti cheat, lots of programs run amazingly well thanks to valve and their contributions to proton and Linux as a whole.

**Reply by Ast3risk-ops** [reply]
I haven't had any issue myself either (I use X11), I've gone back to Linux (this post is a bit old), but I feel my point still stands. I know someone had said in response to this that "freedesktop was destroying Linux," but I think we still have a conundrum that needs fixing.

I think Linux not working compared to Windows was a bit of an overstatement, I wrote this while I was pissed. Linux does work but it has problems, is what I was trying to say.

**Reply by Ast3risk-ops** [reply]
P.S. I have no idea how you found this site.

**Reply by liluzibird** [reply]
I don't understand all the Wayland hype. For something so futuristic, and it can't even support [diagonal monitors](https://sprocketfox.io/xssfox/2021/12/02/xrandr/).

I am also making a [blog](https://userliluzibert.github.io/) using the HBS template. Found your example by looking through the repos that also use the HBS github. Don't know what I plan on doing it, but I'll let the ideas come to me.

**Reply by Ast3risk-ops** [reply]
Yeah, I've tinkered with the theme a bit, it has a lot of features.

**Reply by Ast3risk-ops** [reply]
Set up the repository widget just now, not sure if Hugo gitInfo supports github pages (I use cloudflare pages, which it supports),

**Reply by liluzibird** [reply]
That sounds interesting. How do I add that widget?

**Reply by Ast3risk-ops** [reply]
see https://hbs.razonyang.com/v1/en/docs/widgets/repository/

enable gitInfo: https://gohugo.io/methods/page/gitinfo/

**Reply by liluzibird** [reply]
Interesting. Can't get it working even though it says it should be supported. Maybe I missed something.

Edit: Well, it kind of works. It shows up weird in the main page, but it's fine in all the other sections. It also doesn't show the commit ID stuff.

![image](https://github.com/Ast3risk-ops/blog/assets/57073805/4b6a9e84-d815-483b-bc4e-1077858e4496)

what I want it to look like

![image](https://github.com/Ast3risk-ops/blog/assets/57073805/2b4b3c4f-9202-4433-b368-49c36105a884)

Edit2: Got it. I was working on a testing version and added `enableGitInfo: true` in my config.yaml but forgot to put it in the final code. Still want to find a way to get rid of the Repository section in the main page though, with it still showing in my /news and /docs sections.

**Reply by Ast3risk-ops** [reply]
Yeah, you can probably do that by using some custom CSS.
